enterprise/ha/src/main/java/org/neo4j/kernel/ha/cluster/SwitchToSlave.java

```diff
@@ -367,14 +366,18 @@ void checkDataConsistency( MasterClient masterClient, NeoStoreDataSource neoData
         }
     }
 
-    private void checkMyStoreIdAndMastersStoreId( NeoStoreDataSource neoDataSource, boolean masterIsOld )
+    private void checkMyStoreIdAndMastersStoreId( NeoStoreDataSource neoDataSource, boolean masterIsOld, URI masterUri )
     {
         if ( !masterIsOld )
         {
             StoreId myStoreId = neoDataSource.getStoreId();
 
             ClusterMembers clusterMembers = resolver.resolveDependency( ClusterMembers.class );
-            ClusterMember master = single( filter( inRole( MASTER ), clusterMembers.getMembers() ) );
```

Comment:
It would be useful to explain in the commit message why this change is necessary. Also, this appears to be breaking the PR build.